---
layout:     post
title:      CloVR-16S
date:       14-08-2015 12:00:00
summary:    Analysing microbial community compositions with CloVR-16S

---

While getting myself familiarised with analysing [metagenomic data](https://en.wikipedia.org/wiki/Microbiota) I've been
making my way through some of the excellent walkthroughs available from the
[NIH human microbiome project](http://hmpdacc.org/resources/tools_protocols.php). CloVR
can do many different clever things, but to begin with I'm interested in
analysing the types of microbes that live in the gut and differences that might
occur between healthy subjects and controls. To do this I planned to use the
CloVR-16S pipeline - unfortunately I hit a snag.

CloVR-16S would not let me set the `CloVR 16S Metadata File` despite my best
efforts. After some hours of searching I found an
[old email that fixed my problem](http://lists.igs.umaryland.edu/pipermail/clovr-users/2011-August/000150.html). The
`nuc_FASTA` file containing the sequence data to be analysed wasn't associated
with my `clovr_16s_metadata_file`. Associating the files via the web UI
doesn't work.

Using the "Single FASTA dataset & mapping file" example available
[directly on the CloVR website](http://clovr.org/methods/clovr16s-v1-1-walkthrough/):
click on `CloVR16S_Test_FASTA` and add the tag `clovr_16s_metadata_file`. In the
value field type the name of your metadata file (e.g. `CloVR16S_Test_mapping` -
and make
sure to press save changes). When adding the metadata file add it as 16s metadata file *not* as a
metagenomics mapping file, despite what the tutorial says. 

I ran a 16s analysis contrasting the composition of two healthy microbiomes
([data available online at the human microbiome project](http://hmpdacc.org/HMASM/)). Unsurprisingly
they're broadly similar, but it's a nice proof of concept.
![phylum histogram](/img/phylum-histogram.png)
